src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/editor/Workspace.kt

```diff
@@ -129,9 +129,7 @@ class Workspace(private val project: Project) {
         val editor = editorForFileUrl(jobKey.fileUrl)
 
         if (editor != null) {
-            if (editor.document.modificationStamp == jobKey.modificationTS) {
-                showReport(testReport, editor)
-            }
+            showReport(testReport, editor)
```

Comment:
removing this check would lead to invalid coverage, should the user modify their code while evosuite runs

Reply:
Indeed, but it appears that the way intellij tracks modification stamps is inconsistent. Once we save the file, intellij changes the modification stamp (although the content has not changed) and because of that there was a bug causing coverage not to appear.

Reply:
modification timestamps are indeed very odd.